docs/config.md

```diff
@@ -122,18 +124,304 @@ model_names: <comma-delimited-string|list>
+
+In addition to the mode discussed above, you provide multiple values in
+the objectives key. For example:
+
+```yaml
+objectives:
+- perf_latency
+- perf_throughput
+```
+
+The above config will multiply obtained throughput and latency for each
+measurement by 0.5 (i.e. 0.5 * latency + 0.5 * throughput) and use this as
+the "score" to sort the measurements according to. This mode can be useful
+if you have a more advanced criteria upon which you want to sort the results.
+
+An extension of the above `objectives` is explicitly specifying the weights.
+For example:
+```yaml
+objectives:
+    perf_latency: 10
+    perf_throughput: 15
+```
+
+The score for each measurement will be a weighted average using the weights specified here.
+
+Similar to `<constraint>`, `<objective>` can be specified both globally and on a
+per model basis.
+
+## `<parameter>`
+
+This type can contain `concurrency` or `batch_sizes` as the possible keys. You
+can specify both or one of them. `<parameter>` must be specified under the `model_names`
+key for each model and cannot be specified globally.
+
+An example `<parameter>` looks like below:
+
+```yaml
+parameters:
+    concurrency:
+        start: 2
+        stop: 64
+        step: 8
+    batch_sizes: 1,2,3
+```
+
+A complete config example looks like below:
+
+```yaml
+model_repository: /path/to/model-repository
+model_names:
+  model_1:
+    parameters:
+        concurrency:
+            start: 2
+            stop: 64
+            step: 8
+        batch_sizes: 1,2,3
+```
+
+## `<model-config-parameters>`
+
+This field represents the values that you want to change or sweep through
+using Model Analyzer. All the values supported in the [Triton
+Config](https://github.com/triton-inference-server/server/blob/master/docs/model_configuration.md)
+can be specified or swept through here. `<model-config-parameters>`  should be
+specified on a per model basis and cannot be specified globally (like `<parameter>`).
+
+An example `<model-config-parameters>` look like below:
+```yaml
+model_config_parameters:
+    max_batch_size: [6, 8]
+    dynamic_batching:
+        preferred_batch_size: [[1], [2], [3]]
+        max_queue_delay_microseconds: [200, 300]
+    instance_group:
+    -
+        kind: KIND_GPU
+        count: [1, 2]
+```
+
+Note that for values that accept a list by default you need to specify one
+additional list if you want to sweep through it. Otherwise, it will be only
+used to change the original model config to the value specified and it will
+not sweep through it. You can look at the `preferred_batch_size` as an
+example. This parameter is useful for manual config search. If you are
+interested in the automatic config search, you can take a look at the [Config
+Search](./config_search.md) docs.
+
+A complete YAML config looks like below:
+```yaml
+model_repository: /path/to/model/repository/
+
+run_config_search_disable: True
+model_names:
+  model_1:
+    model_config_parameters:
+        max_batch_size: 2
```

Comment:
Can you create table for all possible model_config_parameters that can be swept through?  Also, do these have a multiplying affect on the number of tests with the `<parameters>` above? If so, you may want to mention this. Otherwise some people may question why Model Analyzer is running 1000+ configurations and taking 2 days to complete.

Reply:
Yes, it has a multiplying effect.

Reply:
I mentioned that all the Triton config parameters can be changed in `<model-config-parameters>`. I'll add a list of common parameters there.